src/App.vue

```diff
@@ -42,7 +42,7 @@
 						{{ t('social', '.well-known/webfinger isn\'t properly set up!') }}
 					</h3>
 					<p v-if="!serverData.checks.checks.wellknown">
-						{{ t('social', 'Social needs the .well-known automatic discovery to be properly set up. If Nextcloud is not installed in the root of the domain, it is often the case that Nextcloud can\'t configure this automatically. To use Social, the admin of this Nextcloud instance needs to manually configure the .well-known redirects: ') }}<a class="external_link"
+						{{ t('social', 'Social needs the .well-known automatic discovery to be properly set up. If Nextcloud is not installed in the root of the domain, it is often the case that Nextcloud cannot configure this automatically. To use Social, the admin of this Nextcloud instance needs to manually configure the .well-known redirects:') }} <a class="external_link"
```

Comment:
Another occurence of "admin" is here.

Reply:
Oops. I wanted to do too quickly. Thanks.